examples/plot_sleep_staging.py

```diff
@@ -1,24 +1,20 @@
 """
 Sleep staging on the Sleep Physionet dataset
 ============================================
-
```

Comment:
just copy the file 

https://raw.githubusercontent.com/braindecode/braindecode/master/examples/plot_sleep_staging.py

to avoid the diff here

Reply:
ok done